src/components/dashboard/LoginRegister/RegisterInput.js

```diff
@@ -50,19 +50,45 @@ const RegisterInput = () => {
         </Heading>
         <FormControl id="firstName">
           <FormLabel>First Name</FormLabel>
-          <Input name="firstName" value={firstName} onChange={onChangeFirstName} isRequired />
+          <Input
+            name="firstName"
+            placeholder="Enter first name"
+            value={firstName}
+            placeholderonChange={onChangeFirstName}
+            isRequired
+          />
         </FormControl>
         <FormControl id="lastName">
           <FormLabel>Last Name</FormLabel>
-          <Input name="lastName" value={lastName} onChange={onChangeLastName} isRequired />
+          <Input
+            name="lastName"
+            placeholder="Enter last name"
+            value={lastName}
+            onChange={onChangeLastName}
+            isRequired
+          />
         </FormControl>
         <FormControl id="email">
           <FormLabel>Email</FormLabel>
-          <Input type="email" name="email" value={email} onChange={onChangeEmail} isRequired />
+          <Input
+            type="email"
+            name="email"
+            placeholder="Enter email address"
+            value={email}
+            onChange={onChangeEmail}
+            isRequired
+          />
         </FormControl>
         <FormControl id="password">
           <FormLabel>Password</FormLabel>
-          <Input type="password" name="password" value={password} onChange={onChangePass} isRequired />
+          <Input
+            type="password"
+            placeholder="8+ characters"
+            name="password"
+            value={password}
+            onChange={onChangePass}
+            isRequired
```

Comment:
```suggestion
            isRequired
            minLength={8}
```

apparently this attribute exists, https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/text, may as well add it while we're at it